lib/rubocop/cop/layout/space_inside_parens.rb

```diff
@@ -86,6 +119,14 @@ def each_extraneous_space(tokens)
           end
         end
 
+        def each_extaneus_space_between_right_parents(token1, token2)
+          return unless right_parens?(token1, token2)
+
+          return if range_between(token1.end_pos, token2.begin_pos).source != ' '
```

Comment:
You should extract the `range_between` call into a variable rather than call it twice.